packages/teuchos/core/src/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ SET(HAVE_TEUCHOSCORE_CXX11 ON)
 SET(YAML_CPP_DEF 0)
```

Comment:
Where is this variable `YAML_CPP_DEF` being used?  I don't see where it is being used anywhere else in this CMake code.

Reply:
I don't see it used anywhere in Trilinos--it is likely a vestige from when yaml-cpp was used before. I will remove it in my next commit.